user_tools/src/spark_rapids_tools/cmdli/tools_cli.py

```diff
@@ -287,6 +288,33 @@ def train(self,
                          wrapper_options=train_args)
         tool_obj.launch()
 
+    def stats(self,
+              config_path: str = None,
+              output_folder: str = None,
+              qual_output: str = None):
+        """The stats cmd generates statistics from the qualification tool output.
```

Comment:
I'd rather this describe what the output is rather then from what its generating stats.  ie outputing stats about each operator/exec. Focus'd on supported/unsupported and how long they ran.